### PR TITLE
Fix/collection view bug

### LINKED ISSUE #19: AutoLayout warning
I've got this message while presenting ImagePicker ViewController:

Warning once only: Detected a case where constraints ambiguously suggest a size of zero for a collection view cell's content view. We're considering the collapse unintentional and using standard size instead.

### PR DESCRIPTION
**Pull request by richardoti**
To fix #19 

I'm still getting these error messages, but at least, there are no bug:
```
2015-08-28 15:11:19.374 MyCubus Staging[6334:179596] the behavior of the UICollectionViewFlowLayout is not defined because:
2015-08-28 15:11:19.374 MyCubus Staging[6334:179596] the item height must be less than the height of the UICollectionView minus the section insets top and bottom values.
2015-08-28 15:11:19.405 MyCubus Staging[6334:179596] the behavior of the UICollectionViewFlowLayout is not defined because:
2015-08-28 15:11:19.406 MyCubus Staging[6334:179596] the item height must be less than the height of the UICollectionView minus the section insets top and bottom values.
```

### CODE REVIEW

File: Source/ImageGallery/ImageGalleryView.swift [hunk 1]
```diff
@@ -126,10 +125,14 @@ public class ImageGalleryView: UIView {
...
 
     topSeparator.frame = CGRect(x: 0, y: 0, width: totalWidth, height: Dimensions.galleryBarHeight)
-    indicator.frame = CGRect(x: (totalWidth - Dimensions.indicatorWidth) / 2, y: (topSeparator.frame.height - Dimensions.indicatorHeight) / 2,
-      width: Dimensions.indicatorWidth, height: Dimensions.indicatorHeight)
-    collectionView.frame = CGRect(x: 0, y: topSeparator.frame.height, width: totalWidth, height: collectionFrame - topSeparator.frame.height)
-    collectionSize = CGSize(width: collectionView.frame.height, height: collectionView.frame.height)
+
+    indicator.center = topSeparator.center
+
+    collectionView.frame = CGRect(x: 0, y: topSeparator.frame.height, width: totalWidth, height: frame.height - topSeparator.frame.height)
```
Comment (RamonGilabert) [inline Source/ImageGallery/ImageGalleryView.swift]: This could be split

File: Source/ImageGallery/ImageGalleryView.swift [hunk 2]
```diff
@@ -126,10 +125,14 @@ public class ImageGalleryView: UIView {
...
+
+    let height = max(collectionView.frame.height, 100)
+    collectionSize = CGSize(width: height, height: height)
+
```
Comment (RamonGilabert) [inline Source/ImageGallery/ImageGalleryView.swift]: I like to have the let before and the properties later. It's consistent through all the Picker.

File: Source/ImagePickerController.swift [hunk 3]
```diff
@@ -174,7 +174,9 @@ public class ImagePickerController: UIViewController {
     let constant = maximum ? GestureConstants.maximumHeight : GestureConstants.minimumHeight
     galleryView.collectionViewLayout.invalidateLayout()
     galleryView.collectionView.frame.size.height = constant - galleryView.topSeparator.frame.height
-    galleryView.collectionSize = CGSize(width: galleryView.collectionView.frame.height, height: galleryView.collectionView.frame.height)
+
+    let height = max(galleryView.collectionView.frame.height, 100)
```
Comment (RamonGilabert) [inline Source/ImagePickerController.swift]: Same here.

File: Source/ImagePickerController.swift [hunk 4]
```diff
@@ -290,7 +292,10 @@ extension ImagePickerController: ImageGalleryPanGestureDelegate {
     if galleryHeight > GestureConstants.minimumHeight {
       galleryView.collectionViewLayout.invalidateLayout()
       galleryView.collectionView.frame.size.height = galleryView.frame.size.height - ImageGalleryView.Dimensions.galleryBarHeight
-      galleryView.collectionSize = CGSize(width: galleryView.collectionView.frame.height, height: galleryView.collectionView.frame.height)
+
+      let height = max(galleryView.collectionView.frame.height, 100)
```
Comment (RamonGilabert) [inline Source/ImagePickerController.swift]: And here.

**Comment by RamonGilabert** [comment]
But it's an opinion, what do you think?

File: Source/ImageGallery/ImageGalleryView.swift (hunk 2 above)
Comment (richardoti) [inline Source/ImageGallery/ImageGalleryView.swift]: I cannot do that, as I am getting `collectionView.frame.height` which has been set a line above.

File: Source/ImageGallery/ImageGalleryView.swift (hunk 1 above)
Comment (richardoti) [inline Source/ImageGallery/ImageGalleryView.swift]: Done

**Comment by richardoti** [comment]
@RamonGilabert I think, it is ready right now.

**Comment by RamonGilabert** [comment]
screen shot 2015-08-28 at 3 55 02 pm

**Comment by RamonGilabert** [comment]
Plus you don't show the images at first.

**Comment by RamonGilabert** [comment]
screen shot 2015-08-28 at 3 55 34 pm

**Comment by richardoti** [comment]
Aah, drop it

**Pull request closed without merging**